README.md

```diff
@@ -3,7 +3,7 @@
 
 # Play Json Ops
 
-Augments the [Play Json library](https://www.playframework.com/documentation/2.5.x/ScalaJson) with some helpful
+Augments the [Play Json library](https://www.playframework.com/documentation/2.6.x/ScalaJson) with some helpful
```

Comment:
Use 2.7.x

Reply:
probably need to update all the other 2.6 references then...?

Reply:
This library cross-compiles to 2.6 as well as 2.7, but we should use the latest version in our documentation.